julia/test/unittest/ndarray.jl

```diff
@@ -1547,8 +1547,8 @@ function test_argmin()
          NaN 2 6]
     x = NDArray(A)
 
-    @test copy(argmin(x, dims = 1)) == [1 2 1]
-    @test copy(argmin(x, dims = 2)) == reshape([1, 2], :, 1)
+    @test copy(argmin(x, dims = 1)) == [2 2 1]
+    @test copy(argmin(x, dims = 2)) == reshape([1, 1], :, 1)
```

Comment:
shouldn't this be [1,2]
previous argmin tests are fine w.r.t numpy

Reply:
I have updated the testcase.